easy-tutorial-step1/README.md

```diff
@@ -1,10 +1,24 @@
-# Easy Tutorial Step 1
+# Easy Extension Framework Tutorial
+
+In this tutorial, we will build a new extension to add new features that do not exist in SAP Commerce Cloud out-of-the-box. As an exercise, we will create an extension to manage delivery slots.
+
+To understand what we will build during this tutorial, you can check the final outcome: [delivery-slot-management](https://github.tools.sap/cx-boosters/easy-extension-samples/tree/main/delivery-slot-management/README.md) extension. You will find in the documentation the list of features.
+
+In this tutorial, you will learn:
+- [How to setup your development environment](#easy-tutorial-step-1---setup-your-development-environment)
+- [How to extend SAP Commerce data model](../easy-tutorial-step2)
+- [How to create and override Spring Beans](../easy-tutorial-step3)
+- [How to create new REST endpoints](../easy-tutorial-step4)
+- [How to create the Backoffice configuration](../easy-tutorial-step5)
+- [How to integrate with SAP Commerce Cloud Composable storefront (aka Spartacus)](../easy-tutorial-step6)
+
+# Easy Tutorial Step 1 - Setup your Development Environment 
 
 ## Overview
-This extension will be used as starting point for a tutorial on the Easy framework
+In this step of the tutorial, you will learn how to setup Easy Extension Framework, install a new extension and setup your IDE.
 
 ## Setup
-Here are the instruction to configure locally the extension and make it works. The extension has been developed to work with the standard electronics-spa accelerator, although it can work easily with other spa accelerators as well with small changes.
+Here are the instructions to configure locally the extension and make it works. The extension has been developed to work with the standard electronics-spa accelerator, although it can work easily with other spa accelerators as well with small changes.
 
 ### Install SAP Commerce Platform
```

Comment:
Link the installation step to the Easy documentation URL: i.e.

https://sap.github.io/easy-extension-framework/getting-started/